WireLogging/Sources/WireLogging/AggregatedLogger.swift

```diff
@@ -31,10 +32,6 @@ final class AggregatedLogger: LoggerProtocol {
 
     // MARK: - LoggerProtocol
 
-    var logFiles: [URL] {
-        loggers.reduce(into: []) { $0 += $1.logFiles }
-    }
-
```

Comment:
Not needed anymore since the only type implementing this was `CocoaLumberjackLogger` and it basically just listed all contents of the `logsDirectory` argument we are now passing in.